mdata/notifierKafka/notifierKafka.go

```diff
@@ -80,11 +80,8 @@ func (c *NotifierKafka) start() {
 		}
 		startOffset, err := c.client.GetOffset(topic, partition, offsetTime)
 		if err != nil {
-			log.Fatalf("kafka-cluster: failed to get offset %d: %s", offsetTime, err)
-		}
-		if startOffset < 0 {
-			// happens when OffsetOldest or an offsetDuration was used and there is no message in the partition
-			startOffset = 0
```

Comment:
I confirmed, I don't think we want to set this to `0` because it would just result in an error from sarama: https://github.com/Shopify/sarama/blob/master/consumer.go#L397

Reply:
it looks like I had not clicked the 'submit review' yesterday...